lib/vehicle.rb

```diff
@@ -5,14 +5,24 @@ class Vehicle
               :year,
               :make,
               :model,
-              :engine
+              :engine,
+              :registration_date,
+              :plate_type
 
   def initialize(vehicle_details)
     @vin = vehicle_details[:vin]
     @year = vehicle_details[:year]
     @make = vehicle_details[:make]
     @model = vehicle_details[:model]
     @engine = vehicle_details[:engine]
+    @registration_date = nil
+    @plate_type = :regular
```

Comment:
This works, but I would like to see this abstracted into it's own method. We want to keep the initialize as clean as possible.